create_db.py

```diff
@@ -38,11 +38,9 @@ def create_default_user(email, password):
     parser = argparse.ArgumentParser()
     parser.add_argument("email", nargs='?', help="The email for super_admin.", default='')
     parser.add_argument("password", nargs='?', help="The password for super_admin.", default='')
-    parser.parse_args()
-    email = parser.email if hasattr(parser, "email") else ''
-    password = parser.password if hasattr(parser, "password") else ''
+    parse = parser.parse_args()
     with current_app.app_context():
         db.create_all()
         stamp()
-        create_default_user(email, password)
+        create_default_user(getattr(parse, "email", ""), getattr(parse, "password", ""))
```

Comment:
Wouldn't it be better to use the Python 3 way of [argparse](https://docs.python.org/3/howto/argparse.html) here and use `if parse.email`? E.g.: 
```python
import argparse
parser = argparse.ArgumentParser()
parser.add_argument("--verbosity", help="increase output verbosity")
args = parser.parse_args()
if args.verbosity:
    print("verbosity turned on")
```

Reply:
I think what happens is that when there is no argument it comes up with error.

Reply:
It seems `getattr(object, attr, default_val)` is the right way to go.